firstvoices/backend/models/widget.py

```diff
@@ -0,0 +1,149 @@
+import rules
+from django.db import models
+from django.utils.translation import gettext as _
+
+from backend.models.base import (
+    BaseControlledSiteContentModel,
+    BaseModel,
+    BaseSiteContentModel,
+)
+from backend.permissions import predicates
+
+
+class WidgetTypes(models.IntegerChoices):
+    # enum intentionally has gaps to allow future changes to keep sequential order
+    WIDGET_ALPHABET = 0
+    WIDGET_APPS = 10
+    WIDGET_CONTACT = 20
+    WIDGET_GALLERY = 30
+    WIDGET_IFRAME = 40
+    WIDGET_KEYBOARDS = 50
+    WIDGET_LOGO = 60
+    WIDGET_LIST = 70
+    WIDGET_QUOTES = 80
+    WIDGET_STATS = 90
+    WIDGET_TEXT = 100
+    WIDGET_TEXTCONCISE = 110
+    WIDGET_TEXTFULL = 120
+    WIDGET_TEXTICONS = 130
+    WIDGET_TEXTMULTI = 140
+    WIDGET_WOTD = 150
+
+
+class WidgetFormats(models.IntegerChoices):
+    # enum intentionally has gaps to allow future changes to keep sequential order
```

Comment:
This is fine, but I'm not sure this one really needs gaps since there isn't really any meaning in sorting these descending vs ascending